pkg/sippyserver/server.go

```diff
@@ -804,6 +807,27 @@ func (s *Server) jsonJobRunRiskAnalysis(w http.ResponseWriter, req *http.Request
 				"message": fmt.Sprintf("error decoding prow job run json in request body: %s", err)})
 			return
 		}
+
+		// validate the jobRun isn't empty
+		// valid case where test artifacts are not available
+		// we want to mark this as a high risk
+		if (jobRun == &models.ProwJobRun{} || &jobRun.ProwJob == &models.ProwJob{} || jobRun.ProwJob.Name == "") {
+
+			log.Warn("Invalid ProwJob provided for analysis, returning elevated risk")
+			result := apitype.ProwJobRunRiskAnalysis{
+				OverallRisk: apitype.FailureRisk{
+					Level:   apitype.FailureRiskLevelHigh,
+					Reasons: []string{"Invalid ProwJob provided for analysis"},
```

Comment:
Could you just add in a little more detail on why it was invalid, specifically that no test results were provided.

Reply:
Sure, in this case the complete ProwJobRun is empty so it is really the fact that we don't have a Name at this point.  I can add checks & specify that the ProwJobRun is empty or that the Name is missing.

Reply:
Ahh I see I was thinking it was just a lack of tests, this is a little more serious. I gotcha. Anything as crisp as possible is good though.